src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -67,15 +71,26 @@ public static SusiThought  scrapeQuora(String profile) {
 		JSONObject quoraProfile = new JSONObject();
 
 		Document userHTML = null;
-		String url = "https://www.quora.com/profile/" + profile;
+		String url = QUORA_URL_BASE + "/profile/" + profile;
 
 		try {
-			userHTML = Jsoup.connect(url).get();
```

Comment:
@SarishiNoHara Some times Jsoup is unable to download the entire webpage. The above change is done to avoid this problem.

http://jmchung.github.io/blog/2013/10/25/how-to-solve-jsoup-does-not-get-complete-html-document/